sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/kafka/jaas/Jaas.java

```diff
@@ -0,0 +1,76 @@
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+package com.azure.spring.cloud.autoconfigure.implementation.kafka.jaas;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.azure.spring.cloud.autoconfigure.implementation.kafka.AzureKafkaConfigurationUtils.AZURE_CONFIGURED_JAAS_OPTIONS_KEY;
+import static com.azure.spring.cloud.autoconfigure.implementation.kafka.AzureKafkaConfigurationUtils.AZURE_CONFIGURED_JAAS_OPTIONS_VALUE;
+
+public class Jaas {
```

Comment:
I know Kafka is using Jaas, but Jaas is a concept not coupled with kafka.
how about move this to `com.azure.spring.cloud.autoconfigure.implementation.jaas` package?